### DISCUSSION TITLE
Documentation and tutorial on REST API not accessible

### DISCUSSION THREAD

**Discussion by MichaelGeFr** [openego/oeplatform#1633] [Feedback]
Unfortunately the tutorial website for the REST API of the OEP is not accessible and I cannot find the correct link to the readthedocs documentation of the API. 

Could you please fix that or provide the original readthedocs link on the main readme?

**Comment by jh-RLI** [comment]
We are on it, but it will take some time :/ Please see our [tutorials/courses](https://openenergyplatform.github.io/academy/tutorials/01_api/01_api_download/) for now. I will work on it by September this year, after that there will be a up-to-date documentation. If you have specific questions, I can help you out.
